graphql-schemas/schema.yml

```
@@ -3040,6 +3040,7 @@ confs:
 - name: SaasResourceTemplateTargetPromotion_v1
   fields:
   - { name: auto, type: boolean }
+  - { name: reDeployOnPublisherConfigChange, type: boolean }
```

apologies for the late review, I've just seen this...

I would go for `redeploy` instead of `reDeploy` since redeploy is actually a word: https://www.merriam-webster.com/dictionary/redeploy

```suggestion
  - { name: redeployOnPublisherConfigChange, type: boolean }
```

---

Fixed